Automatic UI Generator/gadget.xml

```diff
@@ -0,0 +1,469 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Module>
+<ModulePrefs title="Automatic UI Generator" author_email="hagop.jamkojian@epfl.ch" author="Hagop Jamkojian" height="600" width="800">
+<Require feature="dynamic-height"/>
+</ModulePrefs>
+<Content type="html">
+<![CDATA[
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Automatic Gadget Generator</title>
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"/>  
+
+    <script type="text/javascript" src="https://code.jquery.com/jquery-1.7.2.js"></script>
+    <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/headjs/1.0.3/head.min.js"></script>
+    <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/socket.io/1.4.5/socket.io.min.js"></script>
+    <script type="text/javascript" src="https://raw.githubusercontent.com/eligrey/FileSaver.js/master/FileSaver.min.js"></script>
+    <style>
+        #mainDiv {
+            font-size: 16px;
+            line-height: 1.42857143;
+            color: #ffffff;
+            background-color: #5050d2;
+            font-family:arial,sans-serif;
+            margin-bottom:20px;
+            padding-left: 70px;
+            padding-right: 50px;
+            color: white;
+            width: 800px ;
+            height: 600px;
+        }
+        .btn {
+            background-color: white;
+            color: black;
+        }
+        input{
+            color: black;
+        }
+    </style>
+</head>
+<body>
+    <div id="mainDiv">
+        <p></p><br><p></p><br>
+        IP Address: &nbsp; <input type="text" maxlength="15" size="15" id="ipAddress"/><br><p></p>
+        Port Number: &nbsp; <input type="text" maxlength="5" size="5" id="port"/><p></p><br>
+        <button class="btn" background-color="white" id="configurator">Get Configurations</button>
+        <p></p><br/>
+        <div id="experimentsDiv"></div><br>
+        <button class="btn" background-color="white" id="generator">Generate Gadget</button>
+        <script>
+            $('#generator').hide();
+        </script>
+    </div>
+
+    
+    <script>
+        head.ready(function () {
+            var ip;
+            var port;
+            var ws;
+            var body = '<body>\n';
+            var actuators = '';
+            var onOpen = 'ws.onopen = function() { \n connected = 1; \n';
```

Comment:
@hagopj13 what are the \n for when it is inline with js code and not the html styling?